utils/nctl/sh/scenarios/gov96.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# stalling consensus by stopping all but one node. It
+# then restarts the nodes and checks for the chain
+# to progress.
+#
+# Test requested by one of the clients who wants to keep network operational
+# (answering queries) but not producing new blocks and resume it weeks later.
+#
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: gov96"
+    log "------------------------------------------------------------"
+
+    # 0. Wait for network start up
+    do_await_genesis_era_to_complete
+    # 1. Allow chain to progress
+    do_await_era_change
+    # 2. Verify all nodes are in sync
+    check_network_sync
+    # 3-5. Stop three nodes
+    do_stop_node "5"
+    do_stop_node "4"
+    do_stop_node "3"
+    do_stop_node "2"
+    # 6. Ensure chain stalled
+    # Eras are less than a minute long. 
+    # We want to make sure that consensus handles being resumed,
+    # even after being "paused" for multiples of era length.
+    assert_chain_stalled "300"
+    # 7. Verify that the running node can still handle queries.
+    assert_node_operational "1"
+    # 8-10. Restart three nodes
```

Comment:
```suggestion
    # 8-10. Restart four nodes
```
… and "9 - 12"?

Reply:
Fixed in [`070855e` (#1853)](https://github.com/casper-network/casper-node/pull/1853/commits/070855e155d41077944491d7afe5debdc3aace4f)